### PR TITLE
[12.0][IMP] fieldservice: Add Google Map

### PR DESCRIPTION
**Pull request by murtuzasaleh**

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@dreispt @max3903  Can you please review this PR.

**Review by max3903 — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]
This LGTM for a start to enabling google map view for fieldservice.

Is anyone working on adding google map views to fsm locations and orders?

**Comment by murtuzasaleh** [comment]
> This LGTM for a start to enabling google map view for fieldservice.
> 
> Is anyone working on adding google map views to fsm locations and orders?

#398

**Comment by max3903** [comment]
@murtuzasaleh I think the module in the settings should be fieldservice_google_map from #398, not web_view_google_map...

**Comment by brian10048** [comment]
Thanks! I just saw it after I posted.

I was going to comment exactly what @max3903 just suggested as well.  And then #398 will depend on web_view_google_map to install it

File: fieldservice/models/res_config_settings.py [hunk 1]
```diff
@@ -57,6 +57,8 @@ class ResConfigSettings(models.TransientModel):
         string='Manage Recurring Orders')
     module_fieldservice_project = fields.Boolean(
         string='Projects and Tasks')
+    module_web_view_google_map = fields.Boolean(
```
Comment (max3903) [inline fieldservice/models/res_config_settings.py]:
```suggestion
    module_fieldservice_googlemap = fields.Boolean(
```

File: fieldservice/views/res_config_settings.xml [hunk 2]
```diff
@@ -291,6 +291,18 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-xs-12 col-md-6 o_setting_box">
+                            <div class="o_setting_left_pane">
+                                <field name="module_web_view_google_map"/>
```
Comment (max3903) [inline fieldservice/views/res_config_settings.xml]:
```suggestion
                                <field name="module_fieldservice_google_map"/>
```

File: fieldservice/views/res_config_settings.xml [hunk 3]
```diff
@@ -291,6 +291,18 @@
...
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_web_view_google_map"
```
Comment (max3903) [inline fieldservice/views/res_config_settings.xml]:
```suggestion
                                <label for="module_fieldservice_google_map"
```

**Review by dreispt — APPROVED** [review]

**Pull request merged**